07.ls_object/display.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+class Display
+  COLUMN_NUMBER = 3
+
+  def initialize(file_stats, params)
+    @file_stats = file_stats
+    @params = params
+  end
+
+  def print_simple_list
+    row_number = (@file_stats.size.to_f / COLUMN_NUMBER).ceil
+
+    max_lengths = @file_stats.each_slice(row_number).map do |file_stat|
+      file_stat.map { |file| file.name.size }.max
+    end
+
+    before_format_display = @file_stats.map(&:name).each_slice(row_number).to_a
+    before_format_display.last.fill(nil, before_format_display.last.size...row_number) unless before_format_display.last.size == row_number
```

Comment:
```suggestion
    last_col = before_format_display.last
    last_col.fill(nil, last_col.size...row_number) unless last_col.size == row_number
```

みたいに変数に入れた方が見やすくなりそうです

Reply:
ご指摘いただいたよう修正しました。